test/v2/integration/trading.ts

```diff
@@ -0,0 +1,45 @@
+import { coroutine as co } from 'bluebird';
+import should = require('should');
+import * as nock from 'nock';
+
+const TestV2BitGo = require('../../lib/test_bitgo');
+
+describe('V2 Trading', function() {
+  let bitgo;
+  let basecoin;
+  let wallets;
+  let wallet;
+  let trading;
+
+  before(co(function *() {
+    nock.restore();
+
+    bitgo = new TestV2BitGo({ env: 'dev', microservicesUri: 'https://app.bitgo-dev.com' });
+    bitgo.initializeTestVars();
+    basecoin = bitgo.coin('ofc');
+    wallets = basecoin.wallets();
+    basecoin.keychains();
+
+    yield bitgo.authenticateOfcTestUser(bitgo.testUserOTP());
+    wallet = yield wallets.getWallet({ id: TestV2BitGo.V2.OFC_TEST_WALLET_ID });
+    trading = wallet.trading();
+  }));
+
+  it('should create and sign a trade payload', co(function *() {
+    const { payload, signature } = yield trading.signTradePayload({
+      currency: 'btc',
+      amount: '100000000',
+      otherParties: ['test_counterparty_1'],
+      walletPassphrase: TestV2BitGo.OFC_TEST_PASSWORD
+    });
+
+    should.exist(payload);
+    // The payload should be a valid JSON object
+    // NOTE: we shouldn't do any more validation than this, as the schema is subject to change often
+    (() => { JSON.parse(payload); }).should.not.throw();
+
+    should.exist(signature);
+    // signature should be a hex string
+    signature.should.match(/[0-9a-f]+/);
```

Comment:
needs ^ and $, otherwise this will match any string that contains a single `[0-9a-f]`

Reply:
good catch, fixed